cdm/src/test/java/thredds/client/catalog/TestResolve.java

```diff
@@ -33,22 +33,15 @@
 package thredds.client.catalog;
 
 import org.junit.Test;
-import org.junit.experimental.categories.Category;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import ucar.nc2.Attribute;
-import ucar.nc2.NetcdfFile;
-import ucar.nc2.dataset.NetcdfDataset;
-import ucar.unidata.util.test.TestDir;
-import ucar.unidata.util.test.category.NeedsExternalResource;
 
 import java.io.IOException;
 import java.lang.invoke.MethodHandles;
 import java.util.List;
 
-/** Test reletive URL resolution. */
-
-public class TestResolve1 {
+/** Test relative URL resolution. */
+public class TestResolve {
   private static final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
 
   String base="http://www.unidata.ucar.edu/";
```

Comment:
I suppose the tests that use `base` are a good check to see if we can handle a redirect. Would it be better to change `base` to use https, and only do one test with the non-https url, or maybe parameterize this test to run it with both http and https? I'm probably overcomplicating things here.

Reply:
As far as I can tell, we never connect to the URL; it's used only for string comparison.

Reply:
I though I saw a connection in there, but nope, just good ol' string comparisons. My bad.